modules/cnf-scheduling-numa-aware-workloads.adoc

```diff
@@ -129,7 +129,23 @@ Events:
 Deployments that request more resources than is available for scheduling will fail with a `MinimumReplicasUnavailable` error. The deployment succeeds when the required resources become available. Pods remain in the `Pending` state until the required resources are available.
 ====
 
-. Verify that the expected allocated resources are listed for the node. Run the following command:
+. Verify that the expected allocated resources are listed for the node.
+
+.. Identify the node that is running the deployment pod by running the following command, replacing <namespace> with the namespace you specified in the `Deployment` CR:
```

Comment:
@kquinn1204 These cherry picks aren't correct. They should only be changing the image on line 70. But instead, they're also modifying line 132 and adding a whole new step below.

Line 132 was in conflict just because of a whitespace change, so you should just choose the existing content in that case, and not bring in any new content.

When you do cherry picks, it's a good idea to check the diffs of them against the original PR to make sure you're not introducing additional changes

Reply:
my bad @bergerhoffer I did read that section and thought it was an elaboration on what was there already so I added it, but that is not the correct thing to do with a CP. I will revert and only include change needed. I will need to raise a separate PR anyway as something looks off with that guidance `Run the following command, replacing <node_name> with the name of that node that is running the deployment pod. ` there is no <node_name> in the command that is underneath.